dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -542,9 +546,12 @@ public void changeDockerfiles(Namespace ns,
             if (content == null) {
                 log.info("No Dockerfile found at path: '{}'", pathToDockerfile);
             } else {
-                modifyOnGithub(content, gitForkBranch.getBranchName(), gitForkBranch.getImageName(), gitForkBranch.getImageTag(),
-                        ns.get(Constants.GIT_ADDITIONAL_COMMIT_MESSAGE), ns.get(Constants.IGNORE_IMAGE_STRING));
-                isContentModified = true;
+                if (modifyOnGithub(content, gitForkBranch.getBranchName(),
+                        gitForkBranch.getImageName(), gitForkBranch.getImageTag(),
+                        ns.get(Constants.GIT_ADDITIONAL_COMMIT_MESSAGE),
+                        ns.get(Constants.IGNORE_IMAGE_STRING))) {
+                    isContentModified = true;
```

Comment:
```
isContentModified = modifyOnGithub(content, gitForkBranch.getBranchName(),
                        gitForkBranch.getImageName(), gitForkBranch.getImageTag(),
                        ns.get(Constants.GIT_ADDITIONAL_COMMIT_MESSAGE),
                        ns.get(Constants.IGNORE_IMAGE_STRING)));
```
You don't need the conditional here.

Reply:
This will not work - the modifyOnGithub will be called for all GHContent. In this case, the `isContentModified` will have the value returned for the last content on that loop. However, the idea is that the flag should be set to `true` when any content is modified.

Reply:
Got it.  The loop wasn't displayed.  You can aggregate with logical or.  See https://stackoverflow.com/questions/2486472/shortcut-or-assignment-operator-in-java .  According to the documentation `|=` should evaluate both operands so you don't have to worry about shortcutting.
```
isContentModified |= modifyOnGithub(content, gitForkBranch.getBranchName(),
                        gitForkBranch.getImageName(), gitForkBranch.getImageTag(),
                        ns.get(Constants.GIT_ADDITIONAL_COMMIT_MESSAGE),
                        ns.get(Constants.IGNORE_IMAGE_STRING)));
```